packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -312,48 +312,138 @@ class MinimumTextContrastGuideline extends AccessibilityGuideline {
       || paintBounds.right > (window.physicalSize.width * window.devicePixelRatio)  + 50.0;
   }
 
-  List<int> _subsetToRect(ByteData data, Rect paintBounds, int width, int height) {
-    final int newWidth = paintBounds.size.width.ceil();
-    final int newHeight = paintBounds.size.height.ceil();
-    final int leftX = paintBounds.topLeft.dx.ceil();
-    final int rightX = leftX + newWidth;
-    final int topY = paintBounds.topLeft.dy.ceil();
-    final int bottomY = topY + newHeight;
-    final List<int> buffer = <int>[];
-
-    // Data is stored in row major order.
-    for (int i = 0; i < data.lengthInBytes; i+=4) {
-      final int index = i ~/ 4;
-      final int dx = index % width;
-      final int dy = index ~/ width;
-      if (dx >= leftX && dx <= rightX && dy >= topY && dy <= bottomY) {
-        final int r = data.getUint8(i);
-        final int g = data.getUint8(i + 1);
-        final int b = data.getUint8(i + 2);
-        final int a = data.getUint8(i + 3);
-        final int color = (((a & 0xff) << 24) |
-          ((r & 0xff) << 16) |
-          ((g & 0xff) << 8)  |
-          ((b & 0xff) << 0)) & 0xFFFFFFFF;
-        buffer.add(color);
+  @override
+  String get description => 'Text contrast should follow WCAG guidelines';
+}
+
+/// A guideline which verifies that all elements specified by a [Finder] [finder]
```

Comment:
```suggestion
/// A guideline which verifies that all elements specified by a [finder]
```

Reply:
Done; changed to `… specified by [finder]`